kuma/javascript/src/components/section-link.jsx

```diff
@@ -0,0 +1,22 @@
+import * as React from 'react';
+import { render } from 'react-dom';
+
+import { gettext } from '../l10n.js';
+
+import Anchor from '../icons/anchor.svg';
+
+/**
+ * Returns a section anchor element with a nested SVG icon
+ * @param {Object} heading - The current heading DOM node
+ * @returns section anchor element as an HTML element
+ */
+export default function sectionAnchor(heading) {
+    const anchor = document.createElement('a');
+    anchor.href = `#${heading.id}`;
+    anchor.classList.add('section-link');
+    // heading.innerText is the current heading and will already be translated
+    anchor.setAttribute('aria-label', gettext('Link to ') + heading.innerText);
```

Comment:
```suggestion
    anchor.setAttribute('aria-label', `${gettext('Link to')} ${heading.innerText}`);
```
Otherwise the localizers would have to be confronted with the extra whitespace and it might confuse or they might delete it.